src/wmnds/components/inset-text/_example.njk

```diff
@@ -0,0 +1,10 @@
+{% raw %}
+{% from "wmnds/components/inset-text/_inset-text.njk" import wmndsInsetText %}
+
+{{
+    wmndsInsetText({
+        html: Example of inset text,
```

Comment:
Update this to "contentHTML" to be consistent with the naming in link/btn components.